assets/js/editor-components/product-stock-control/index.tsx

```diff
@@ -1,54 +1,67 @@
 /**
  * External dependencies
  */
-import CheckboxList from '@woocommerce/base-components/checkbox-list';
+import { __, sprintf } from '@wordpress/i18n';
 import { getSetting } from '@woocommerce/settings';
-import { useCallback, useState } from '@wordpress/element';
+import { useCallback, useState, useEffect } from '@wordpress/element';
+import { useDebounce } from 'use-debounce';
+import { ToggleControl } from '@wordpress/components';
 
 export interface ProductStockControlProps {
 	value: Array< string >;
 	setAttributes: ( attributes: Record< string, unknown > ) => void;
 }
 
+// Should out of stock items be hidden globally?
```

Comment:
Is this a follow up todo?

Reply:
never mind I see this code was moved from below. Question persists, just that you probably don't have an answer.